src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -38,6 +79,41 @@ class AssetsTable extends React.Component {
     }
   }
 
+  onSortClick(columnKey) {
+    // const prevColumnKey = this.state.sortState.columnKey;
+    const columns = [...this.state.tableColumns];
+    const column = this.state.tableColumns.find(col => (col.key === columnKey));
+    let newDirection;
+
+    if (this.state.sortState.columnKey === columnKey) {
+      newDirection = this.state.sortState.direction === 'descending' ? 'ascending' : 'descending';
+      column.sortDirection = newDirection;
+
+      // then we want to make the appropriate API call
+    } else {
+      // is this the best way to do a line break?
+      const oldColumn = this.state.tableColumns.find(col =>
+        (col.key === this.state.sortState.columnKey));
+
+      // can we have a better way to do this that's less ambiguous than relying on the default?
+      oldColumn.sortDirection = '';
+      newDirection = 'ascending';
+      column.sortDirection = newDirection;
+
+      // then we want to make the appropriate API call
+      // who should handle this? AssetsTable? What happens if a user has a filter selected and a sort? Does this integrate
+      // with what's happening in AssetsPage? I need to look into this further.
```

Comment:
I need to consider this!

Reply:
Good question! Having AssetsTable in charge seems like a good initial solution, but we should probably discuss as a team.